docs/manual/src/docs/asciidoc/_includes/servlet/oauth2/oauth2-resourceserver.adoc

```diff
@@ -1456,39 +1464,40 @@ security:
         client-id: client
         client-secret: secret
 ----
+====
 
 Where `https://idp.example.com/introspect` is the introspection endpoint hosted by your authorization server and `client-id` and `client-secret` are the credentials needed to hit that endpoint.
 
-Resource Server will use these properties to further self-configure and subsequently validate incoming JWTs.
-
-[NOTE]
-When using introspection, the authorization server's word is the law.
-If the authorization server responses that the token is valid, then it is.
+Resource Server uses these properties to further self-configure and subsequently validate incoming JWTs.
 
-And that's it!
+NOTE: When using introspection, the authorization server's word is law.
+If the authorization server responsed that the token is valid, it is.
 
 ==== Startup Expectations
 
-When this property and these dependencies are used, Resource Server will automatically configure itself to validate Opaque Bearer Tokens.
+When this property and these dependencies are used, Resource Server automatically configures itself to validate Opaque Bearer Tokens.
 
-This startup process is quite a bit simpler than for JWTs since no endpoints need to be discovered and no additional validation rules get added.
+This startup process is quite a bit simpler than for JWTs, since no endpoints need to be discovered and no additional validation rules get added.
 
 ==== Runtime Expectations
 
-Once the application is started up, Resource Server will attempt to process any request containing an `Authorization: Bearer` header:
+Once the application has started, Resource Server tries to process any request containing an `Authorization: Bearer` header:
 
-```http
+====
+[source,text]
```

Comment:
I think this should be `[source,http]` to give better formatting.

Reply:
OK. Done.

Reply:
OK